packages/cli/lib/utils.js

```diff
@@ -386,8 +388,96 @@ exports.getRepositoryFileName = function(repositoryName) {
   return `${_.kebabCase(repositoryName)}.repository.ts`;
 };
 
+/**
+ * Returns the serviceName in the directory file format for the service
+ * @param {string} serviceName
+ */
+exports.getServiceFileName = function(serviceName) {
+  return `${_.kebabCase(serviceName)}.service.ts`;
+};
+
+/**
+ *
+ * Returns the connector property for the datasource file
+ * @param {string} datasourcesDir path for sources
+ * @param {string} dataSourceClass class name for the datasoure
+ */
+exports.getDataSourceConnectorName = function(datasourcesDir, dataSourceClass) {
+  if (!dataSourceClass) {
+    return false;
+  }
+  let result;
+  let jsonFileContent;
+
+  let datasourceJSONFile = path.join(
+    datasourcesDir,
+    dataSourceClass.replace('Datasource', '.datasource.json').toLowerCase(),
+  );
+
+  debug(`reading ${datasourceJSONFile}`);
+  try {
+    jsonFileContent = JSON.parse(fs.readFileSync(datasourceJSONFile, 'utf8'));
+  } catch (err) {
+    debug(`Error reading file ${datasourceJSONFile}: ${err.message}`);
+    throw new Error(err);
```

Comment:
Don't create a new Error object, it will discard the original stack trace and other useful information from `err`. Just `throw err`.